app/src/main/java/ar/com/wolox/android/example/utils/ViewExtensions.kt

```diff
@@ -14,4 +21,10 @@ fun View.toggleVisibility(visible: Boolean) {
  */
 fun View.togglePresence(present: Boolean) {
     visibility = if (present) View.VISIBLE else View.GONE
-}
\ No newline at end of file
+}
+
+fun String.abbreviationDayFormat(): String {
+    return PrettyTime().format(Date(SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss.SSS'Z'").parse(this).time))
+}
+
+fun ImageView.glideImage(url: String) = Glide.with(this).load(url).placeholder(ColorDrawable(Color.WHITE)).into(this)
```

Comment:
Excellent!
Glide also provides the .error() method, so you can set an image to be rendered when the url has an error.